dali/operators/generic/one_hot.h

```diff
@@ -57,7 +57,20 @@ class OneHot : public Operator<CPUBackend> {
         axis_(spec.GetArgument<int>("axis")),
         output_type_(spec.GetArgument<DALIDataType>(arg_names::kDtype)),
         on_value_(spec.GetArgument<float>("on_value")),
-        off_value_(spec.GetArgument<float>("off_value")) {}
+        off_value_(spec.GetArgument<float>("off_value")),
+        layout_axis_name_(spec.GetArgument<std::string>("layout_axis_name")) {
```

Comment:
FYI: We're gradually departing from storing argument values in constructors. Layouts are likely to remain constant, though.